matchmaker/Assets/ServerObjects/ExperimentSO.cs

```diff
@@ -0,0 +1,24 @@
+using GPF.ServerObjects;
+using System.Collections.Generic;
+
+namespace ServerObjects.Test
+{
+    [DataStorePath("sync.experiment")]
+    [Syncable]
+    [Register("experiment")]
+    public class ExperimentSO : RESTBaseSO
+    {
+        public Dictionary<string, string> experiments = new Dictionary<string, string>();
+
+        public void Handler(RESTRequestMessage msg)
+        {
+            experiments = GetParamsFromQueryString(msg.Uri.Query);
```

Comment:
I recommend handling cases where the request is malformed or supplying invalid data and returning the appropriate [status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses).